packages/offline-packages.yaml.tmpl

```diff
@@ -115,8 +184,8 @@ editions:
               - name: sync_diff_inspector # New in v6.0.0
                 src:
                   type: oci
-                  # version release on master branch.
-                  url: "{{ .Release.registry }}/pingcap/tidb-tools/package:master_{{ .Release.os }}_{{ .Release.arch }}"
+                  # sync_diff_inspector stop builds after v8.5.1 on tidb-tools repo. So we use the latest v8.5.x version for patch version lower then v9.0.0.
+                  "{{ .Release.registry }}/pingcap/tidb-tools/package:v8.5.1_{{.Release.os }}_{{ .Release.arch }}"
```

Comment:
for history patch version we pin the version as v8.5.1 that will be used to compose offline toolkit package.